src/navigate-pop/launching-applications.md

```diff
@@ -0,0 +1,92 @@
+<!--Sources
+Pop Training Guide (Google doc)
+https://support.system76.com/articles/pop-basics/
+
+-->
+# Locate, Launch, and Navigate Between Applications
+
+---
+
+## Browsing Installed Applications
+
+Installed applications can be viewed in the Applications menu and the Pop!\_Shop Installed screen.
+
+### The Applications Menu
+
+The Applications menu can be accessed from the Dock or by clicking `Applications` in the upper left corner of the screen.
+
+![Launch Applications Menu](/images/locate-launch-applications/launch-applications-menus.png)
+
+Applications can be grouped into folders. Click `Create Folder` and give the folder a name.
+
+![Create Application Folder](/images/locate-launch-applications/create-application-folder.png)
+
+Click and drag an application icon to add it to your folder.
+
+<video autoplay loop>
+    <source src="/images/locate-launch-applications/drag-application.webm" />
+</video>
+
+### The Pop!\_Shop
+
+Launch the Pop!\_Shop, then click the `Installed` button.
+
+![Pop Shop Installed Apps](/images/locate-launch-applications/pop-shop-installed-apps.png)
+
+## Launching Applications
+
+Applications can be launched from the Dock, using the Launcher, the Applications menu, and from the Terminal.
```

Comment:
This sentence comes off a little inconsistent.

> Applications can be launched from the Dock, the Launcher, the Applications menu, and the Terminal.

or

> Applications can be launched from the Dock, using the Launcher, from the Applications menu, and from the Terminal.

would be more consistent (either the directional word like "from/using" is part of the list item and each list item needs one, or it's not part of the list and none of the list items should have one.)